TrackletAlgorithm/MatchEngine.cc

```diff
@@ -29,14 +29,15 @@ void MatchEngine(const BXType bx, BXType& bx_o,
 				 const VMStubMEMemory<VMSMEType, NBITBIN>& inputStubData,
 				 const VMProjectionMemory<VMPMEType>& inputProjectionData,
 				 CandidateMatchMemory& outputCandidateMatch) {
+#pragma HLS inline
 	//
 	//Initialize table for bend-rinv consistency
 	//
 	//bool table[(L<4)?256:512]; //FIXME Need to figure out how to replace 256 with meaningful const.
```

Comment:
can this be deleted? looks like it is defunct.

Reply:
What do you mean by "defunct"? The inline shaves off a clock cycle, which is useful since the rewind option doesn't work yet for the ME.

Reply:
delete the commented out line with the fix me

Reply:
Done.